src/assets/logo.svg

```diff
@@ -0,0 +1,24 @@
+</defs>
+</svg>
```

Comment:
This SVG embeds a very large base64 PNG image via data URI. Inlining large base64 assets inflates bundle size, slows down parsing and increases network/JS bundle weight.

**Sugerencia:**
```typescript
Replace the embedded base64 with a reference to an external optimized asset (e.g., /assets/logo.png) or use an optimized SVG-only logo. If embedding is required, compress/optimize the image and document the size impact.
```